scrapy/commands/startproject.py

```diff
@@ -76,3 +74,9 @@ def run(self, args, opts):
         print("You can start your first spider with:")
         print("    cd %s" % project_name)
         print("    scrapy genspider example example.com")
+
+    @property
+    def templates_dir(self):
+        _templates_base_dir = self.settings['TEMPLATES_DIR'] or \
```

Comment:
The above approach has been followed in genspider.py . Should I change that too?